src/Compilers/VisualBasic/Portable/VBResources.Designer.vb

```diff
@@ -9010,6 +9010,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Property
         
         '''<summary>
+        '''  Looks up a localized string similar to An error occurred while writing the output file: {0}.
+        '''</summary>
+        Friend ReadOnly Property ERR_PeWritingFailure() As String
+            Get
+                Return ResourceManager.GetString("ERR_PeWritingFailure", resourceCulture)
+            End Get
+        End Property
```

Comment:
Right but why did you need a new entry in the designer file?  You didn't add a new resource string, you modified the contents of an existing one.  That shouldn't effect the designer file.